python/tests/visualization/test_bloch_sphere_visualization.py

```diff
@@ -0,0 +1,147 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+import os
+import pytest
+import numpy as np
+
+import cudaq
+
+qutip = pytest.importorskip("qutip")
+
+import io
+from contextlib import redirect_stdout
+
+
+@pytest.fixture(autouse=True)
+def do_something():
+    yield
+    cudaq.__clearKernelRegistries()
+
+
+# write sample kernels
+
+
+@cudaq.kernel
+def single_qubit_kernel():
+    # Run any random single qubit code. Currently using the code from examples/python/intro.py
+    qubit = cudaq.qubit()
+    h(qubit)
+    x(qubit)
+    y(qubit)
+    z(qubit)
+    t(qubit)
+    s(qubit)
+
+    mz(qubit)
+
+
+@cudaq.kernel
+def two_qubit_kernel():
```

Comment:
```
@cudaq.kernel
def two_qubit_kernel():
    qvector = cudaq.qvector(2)
    x(qvector[0])
    ry(np.pi / 2., qvector[1])
    mz(qvector)
```

Reply:
I've simplified the kernel as requested.